test/common.py

```diff
@@ -151,13 +152,24 @@ def safeCoalesce(self, t):
 
         return tg
 
+    def unwrapVariables(self, x, y):
+        if isinstance(x, Variable) and isinstance(y, Variable):
+            return x.data, y.data
+        if isinstance(x, Variable) and torch.is_tensor(y):
+            return x.data, y
+        if isinstance(x, Variable) and isinstance(y, Number):
+            return x.data[0], y
+        if torch.is_tensor(x) and isinstance(y, Variable):
+            return x, y.data
+        if isinstance(x, Number) and isinstance(y, Variable):
+            return x, y.data[0]
```

Comment:
@apaszke In case of mixed arguments, would you fail by `raise Exception` or some other way?